file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,349 @@
+      sit2_obj_act_nav     TYPE ty_sit2_obj_act_nav_list,
+      sit2_obj_act_cb      TYPE ty_sit2_obj_act_cb_list,
+    END OF ty_sit2_obj_act,
+
+    "! <p class="shorttext">List of SIT2_OBJ_ACT</p>
+    "! List of SIT2_OBJ_ACT
+    ty_sit2_obj_act_list TYPE STANDARD TABLE OF ty_sit2_obj_act WITH EMPTY KEY,
+
+    "! <p class="shorttext">SIT2_OBJ_CB</p>
+    "! SIT2_OBJ_CB
+    BEGIN OF ty_sit2_obj_cb,
+      "! <p class="shorttext">Object Reusable</p>
+      sitnobjclbkisreusable TYPE sit2_de_reusable,
+      "! <p class="shorttext">Source</p>
+      sitnobjclbkscope      TYPE sit2_de_scope,
+      "! <p class="shorttext">Reference to</p>
+      sitncallbackclassname TYPE classname,
+      "! <p class="shorttext">Callback ID</p>
+      sitncallbackid        TYPE sit2_de_callback_id,
+      sit2_obj_cb_par       TYPE ty_sit2_obj_cb_par_list,
+    END OF ty_sit2_obj_cb,
+
+    "! <p class="shorttext">List of SIT2_OBJ_CB</p>
+    "! List of SIT2_OBJ_CB
+    ty_sit2_obj_cb_list TYPE STANDARD TABLE OF ty_sit2_obj_cb WITH EMPTY KEY,
+
+    "! <p class="shorttext">SIT2_OBJ_VH_S</p>
+    "! SIT2_OBJ_VH_S
+    BEGIN OF ty_sit2_obj_vh_s,
+      "! <p class="shorttext">Object Reusable</p>
+      sitnobjvhsrvcisreusable      TYPE sit2_de_reusable,
+      "! <p class="shorttext">Service Version for Value Help</p>
+      sitnobjvhsrvcversion         TYPE sit2_de_vh_srv_version,
+      "! <p class="shorttext">Service Protocol Version for Value Help</p>
+      sitnobjvhsrvcprotocolversion TYPE sit2_de_vh_srv_protocol_vers,
+      "! <p class="shorttext">Service Path Type for Value Help</p>
+      sitnobjvhsrvcpathtype        TYPE sit2_de_vh_srv_path_type,
+      "! <p class="shorttext">Source</p>
+      sitnobjvhsrvcscope           TYPE sit2_de_scope,
+      "! <p class="shorttext">Service Definition for Value Help</p>
+      sitnobjvhsrvcdefinition      TYPE sit2_de_vh_srv_definition,
+      "! <p class="shorttext">Service Binding for Value Help</p>
+      sitnobjvhsrvcbinding         TYPE sit2_de_vh_srv_binding,
+      "! <p class="shorttext">Service ID for Value Help</p>
+      sitnobjvhsrvcid              TYPE sit2_de_vh_srv_id,
+      "! <p class="shorttext">Service Custom Path for Value Help</p>
+      sitnobjvhsrvccustompath      TYPE sit2_de_vh_srv_custom_path,
+      sit2_obj_vh_s_p              TYPE ty_sit2_obj_vh_s_p_list,
+    END OF ty_sit2_obj_vh_s,
+
+    "! <p class="shorttext">List of SIT2_OBJ_VH_S</p>
+    "! List of SIT2_OBJ_VH_S
+    ty_sit2_obj_vh_s_list TYPE STANDARD TABLE OF ty_sit2_obj_vh_s WITH EMPTY KEY,
+
+    "! <p class="shorttext">SIT2_OBJECT</p>
+    "! SIT2_OBJECT
+    BEGIN OF ty_sit2_object,
+      "! <p class="shorttext">Situation Object ID</p>
+      sitnobjectid          TYPE sit2_de_object_id,
+      "! <p class="shorttext">ABAP Language Version</p>
+      abap_language_version TYPE abap_language_version,
+      "! <p class="shorttext">Object Extensible</p>
+      sitnobjisextensible   TYPE sit2_de_extensible,
```

Comment:
As mentioned before, I suggest to rename the fields. E.g. something like `extensible`?
```suggestion
      extensible   TYPE sit2_de_extensible,
```

Furthermore, it would be great if the types in this interface would be self-contained.